### DISCUSSION TITLE
Is it possible broadcast video to multiple ip address?

### DISCUSSION THREAD

**Discussion by iamfaith** [svpcom/wfb-ng#125] [Questions]
While receiving UDP packets using wfb_rx -c 0.0.0.0, can multiple clients receive the video in the same LAN?

**Comment by svpcom** [comment]
Use 224.0.0.1 multicast address instead

**Reply by iamfaith** [reply]
Thanks for your response. BTW, what's the best option for FEC value with the lowest latency?

**Reply by svpcom** [reply]
There is no simple answer for this  question. FEC parameters depends from radio channel profile. Try to start reading from: https://trs.jpl.nasa.gov/bitstream/handle/2014/34531/94-0881.pdf?sequence=1
